charts/visor/templates/deployment.yaml

```diff
@@ -65,28 +76,95 @@ spec:
       - name: visor
         image: "{{ .Values.image.repo }}:{{ .Values.image.tag }}"
         imagePullPolicy: "{{ .Values.image.pullPolicy }}"
-        args: ["--tracing={{ .Values.jaeger.enabled }}", "run"]
+        args:
+          # Global args
+          # debugging args
+          {{- if and .Values.jaeger .Values.jaeger.enabled }}
+          - "--tracing={{ .Values.jaeger.enabled }}"
+          - "--jaeger-agent-host={{ .Values.jaeger.host }}"
+          - "--jaeger-agent-port={{ .Values.jaeger.port }}"
+          - "--jaeger-service-name={{ .Values.jaeger.serviceName }}"
+          {{- if .Values.jaeger.sampler }}
+          - "--jaeger-sampler-type={{ .Values.jaeger.sampler.type }}"
+          - "--jaeger-sample-param={{ .Values.jaeger.sampler.param }}"
+          {{- end }}
+          {{- end }}
+
+          {{- if .Values.logLevel }}
+          - "--log-level={{ .Values.logLevel }}"
+          {{- end }}
+
+          {{- if .Values.logLevelNamed }}
+          - "--log-level-named={{ .Values.logLevelNamed }}"
+          {{- end }}
+
+          # database args
+          {{- if .Values.pgPoolSize }}
+          - "--db-pool-size={{ int .Values.pgPoolSize }}"
+          {{- end }}
+
+          {{- if .Values.pgAppName }}
+          - "--name={{ .Values.pgAppName }}"
+          {{- end }}
+
+          - "--allow-schema-migration={{ .Values.allowAutomaticMigration }}"
+
+          # lens config
+          {{- if .Values.lens }}
+          {{- if and .Values.lens.lotusAPI }}
+          - "--lens=lotus"
+          {{- $lotusAPIMultiaddr := "/dns/{{ .Release.Name }}-lotus-daemon.{{ .Release.Namespace }}.svc.cluster.local/tcp/1234" -}}
```

Comment:
I don't think we'll ever use this automatic lotus address. I'd prefer to remove it and add a check to ensure that there is a multiaddr set in the values